### PR TITLE
miscellaneous small improvements

### PR DESCRIPTION
**Pull request by DonDiego**
Here are a bunch of small changes that I am carrying locally. Please merge or cherry-pick at your discretion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,11 +178,11 @@ access to the management interface.
 
 ### Configure openvpn-monitor
 
-Copy the example configuration file `openvpn-monitor.conf.example` to the same
+Copy the example configuration file `openvpn-monitor.conf` to the same
 directory as openvpn-monitor.py.
```
Comment (furlongm) [inline README.md:182]: This was added due to a request in https://github.com/furlongm/openvpn-monitor/issues/33

When running from source in a git repo, it makes sense, as a git pull fails if the conf has been modified.

Comment (DonDiego) [reply README.md:182]: But you have to create a copy of the config when running from source in a Git repo. #33 looks like insufficient scripting knowledge to me. It's really not that hard to script "git pull" in a way that handles conflicts.

Anyway, it's your decision. Merge or reject as you please. It's only 1/4 commits anyway.

Comment (DonDiego) [reply README.md:182]: Also, git pull need not always fail if the conf file has been modified. It might also conveniently merge in changes.

**Comment by DonDiego** [comment]
The region information seems to work better for geoip2 databases. I used geoip1 before, but that is deprecated, so I guess you can drop that patch.

**Comment by furlongm** [comment]
I cherry-picked 2/4, and added some other bits (added maps_height to settings - I guess it depends on your monitor size!).

The .example file I feel is good practice, and it's helpful to me to have it as-is. The scripting for `git pull` is going to be slightly more complex vs a simple `cp`.

As you mentioned, geop1 is deprecated and, to be honest, I was looking at alternatives to geoip, or adding the option to have different geoip providers.

Thanks again for the PR! Sorry for the delay in getting to it.

**Pull request closed without merging**